lib/katello/tasks/cv_diff.rake

```diff
@@ -0,0 +1,243 @@
+namespace :katello do
+  #desc <<-DESC.strip_heredoc
+  desc <<-DESC
+    Calculates and shows the difference (in terms of package contents) between two versions of the same Content View.
+
+      Parameters:
+
+          * LEFT=CV:version  : Content View and version identifiers.
+                             : CV can be either a name or label or numerical ID of a Content View.
+                             : Omitting the version is the same as specifying the latest version of this Content View.
+                             : Version is specified as major.minor. See examples below.
+                             : Examples:
+                             :   LEFT="This nice CV"  -- latest version of a Content View named "This nice CV"
+                             :   LEFT="this_nice_cv"  -- latest version of a Content View named or labeled "this_nice_cv"
+                             :   LEFT="This nice CV:15.0"  -- version 15.0 of "This nice CV"
+                             :   LEFT="this_nice_cv:15.0"  -- version 15.0 of "this_nice_cv"
```

Comment:
```suggestion
```
I think this one can be safely removed as it is basically implied by the other examples.